### ISSUE TITLE
Docs: styling worng or outdated

### ISSUE THREAD

**Issue by relic-yuexi** [typst/typst#1946]
https://typst.app/docs/reference/styling/

![image](https://github.com/typst/typst/assets/53668275/afc3f769-fc94-49c7-a9d4-e4e8de58469f)

the code `#set align(center)` didn't work.

And I want to center the first-level headings, but not the other headings, and I want the first-level headings to use "Ⅰ." "Ⅱ." and the second-level headings to use 1.1 1.2, how should I write the code?

**Comment by PgBiel** [comment]
Yeah I think the docs meant to `align(center, block[` (or to have a 100% width block); as is, the block shrinks down to the size of the heading title, and aligning inside it doesn't change anything.

**Comment by Enivex** [comment]
```
#set page("a7")
#set heading(numbering: "(I)")

#show heading: it => {
  set align(center)
  set text(font: "Inria Serif", fallback: false)
  [\~ #emph(it.body)  #counter(heading).display() \~]
}

= Dragon
With a base health of 15, the
dragon is the most powerful
creature.

= Manticore
While less powerful than the
dragon, the manticore gets
extra style points.
```
![image](https://github.com/typst/typst/assets/2191754/fe761aab-ff02-45d3-ae0d-11eaa3a2ad11)

I'm not entirely sure why the block was there in the first place
